content/en/docs/concepts/verification-by-reference.md

```diff
@@ -306,21 +306,14 @@ Another index of type `notary.verification`, which adds the scanning verificatio
 }
 ```
 
-The nv2 client could retrieve all signatures with the following mocked command
+The notation client could retrieve all signatures with the following command
 
 ```shell
-nv2 verification pull registry.contoso.com/marketing/web:a2b2
-nv2 verification pull registry.contoso.com/marketing/web@256:efg123
+notation verify registry.contoso.com/marketing/web:a2b2
+notation verify registry.contoso.com/marketing/web@256:efg123
 ```
 
-or
-
-```shell
-nv2 verify registry.contoso.com/marketing/web:a2b2
-nv2 verify registry.contoso.com/marketing/web@256:efg123
-```
-
-The `nv2 verify` API compares the artifact with local configuration to attest to it matching the configured rules.
+The `notation verify` API compares the artifact with local configuration to attest to it matching the configured rules.
```

Comment:
The description can be changed to "The `notation verify` verifies the artifacts against signatures based on the trust policy. ". Put this sentence before the two examples above.